RIE/RIE271and190.xml

```diff
@@ -183,13 +185,35 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <div type="textpart" subtype="face" xml:id="B" n="1">
                         <lb rend="right-to-left"/>
                     </div>
-                    
+
                     <div type="textpart" n="2" subtype="face" xml:id="C">
                         <ab/>
                     </div>
                 </div>
             </div>
+            <div type="bibliography">
+           <listBibl type="secondary">
+                 <bibl>
+                    <ptr target="bm:RIE1"/>
+                    <citedRange unit="page">268—271</citedRange>
+                    <citedRange unit="page">370—372</citedRange>
+                 </bibl>
+                 <bibl>
+                    <ptr target="bm:RIE2"/>
+                    <citedRange unit="plate">124—128</citedRange>
+                    <citedRange unit="plate">181</citedRange>
+                 </bibl>
+                <bibl>
+                  <ptr target="bm:RIE3A"/>
```

Comment:
I cannot find this tag in the zotero library. does it exist?

Reply:
The correct tag for this in the Zotero library already exists and is bm:BernandRIEIIIA please replace all occurrences